generator/src/test/java/com/scottlogic/deg/generator/cucumber/LessThan.feature

```diff
@@ -2,12 +2,332 @@ Feature: User can specify that a numeric value is lower than, but not equal to,
 
 Background:
      Given the generation strategy is full
-      And there is a field foo
-      And foo is of type "numeric"
-      And foo is anything but null
+       And there is a field foo
+       And foo is of type "numeric"
+       And foo is anything but null
 
+# alone
+@ignore #589: produces big-integer numbers rather than integer values, not sure why
+Scenario: User requires integer data
+     Given foo is less than 100
+       And the generator can generate at most 5 rows
+     Then the following data should be generated:
+       | foo         |
+       | -2147482624 |
+       | -2147482623 |
+       | -2147482622 |
+       | -2147482621 |
+       | -2147482620 |
+
+@ignore #589: produces big-integer numbers rather than integer values, not sure why
+Scenario: User requires decimal data
+     Given foo is less than 100.1
+       And the generator can generate at most 5 rows
+     Then the following data should be generated:
+       | foo           |
+       | -2147482624.0 |
+       | -2147482623.9 |
+       | -2147482623.8 |
+       | -2147482623.7 |
+       | -2147482623.6 |
+
+@ignore #589: produces big-integer numbers rather than integer values, not sure why
+Scenario: User requires negative numbers data
+     Given foo is less than 0
+       And the generator can generate at most 5 rows
+     Then the following data should be included in what is generated:
+       | foo |
+       | -1  |
+
+@ignore #589: produces big-integer numbers rather than integer values, not sure why
+Scenario: User requires 0
+     Given foo is less than 1
+       And the generator can generate at most 5 rows
+     Then the following data should be included in what is generated:
```

Comment:
we're trying to remove "should be included" in favour of being more specific. Can this test be rewritten to allow for  removing this?

Reply:
Yes agreed, will ref #594 and ignore